base/applications/rapps/unattended.cpp

```diff
@@ -366,14 +366,17 @@ ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
             if (hWindow)
             {
                 /* Activate the window in the other instance */
+                ShowWindow(hWindow, SW_SHOW);
                 SwitchToThisWindow(hWindow, TRUE);
                 if (bAppwizMode)
                     PostMessage(hWindow, WM_COMMAND, ID_ACTIVATE_APPWIZ, 0);
 
-                if (hMutex)
-                    CloseHandle(hMutex);
-
-                return FALSE;
+                if (IsWindowVisible(hWindow))
```

Comment:
Why do we need a visibility check here?

Reply:
It's not strictly tied to this PR.

It makes it more robust, make sure we actually made the window in the other process visible (and that the window is still valid in the first place).

Reply:
I don't think it would be too dramatic to keep the original code there. This would mean that if the other instance RAPPS window closes in-between the FindWindow call and here, no window would show up. I don't think it's a huge problem because that other existing window could disappear just after the (hypothetically successful) IsWindowVisible() call, (just a matter of milliseconds or so), and from the user's point of view, when (s)he tries to open RAPPS, the window wouldn't appear as well.